misk/src/main/kotlin/misk/MiskServiceModule.kt

```diff
@@ -97,18 +97,17 @@ class MiskCommonServiceModule : KAbstractModule() {
     val serviceListBinding = injector.getBinding(serviceSetKey)
     val invalidServices = serviceListBinding
         .acceptTargetVisitor(CheckServicesVisitor())
-        .sorted()
-
-    // Confirm all services have been registered as singleton. If they aren't singletons,
-    // _readiness checks will fail
-    check(invalidServices.isEmpty()) {
-      "the following services are not marked as @Singleton: ${invalidServices.joinToString(", ")}"
-    }
+        .sorted().toMutableList()
 
     val builder = ServiceGraphBuilder()
 
     // Support the new ServiceModule API.
     for (entry in serviceEntries) {
+      // Check that the bound ServiceEntry is Singleton. This does the same thing as the visitor for
```

Comment:
This comment probably doesn’t make much sense to future readers of this code?

Reply:
It refers to the stuff that is happening on lines 95-100.

When we eventually remove the multibinder for `Service` (this is planned, right?), this comment can be removed with the code on 95-100 and the `CheckServicesVisitor` class.